docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -2257,6 +2257,34 @@ For more information about Grafana Enterprise, refer to [Grafana Enterprise]({{<
 
 Keys of alpha features to enable, separated by space.
 
+<hr>
+
+## [feature_management]
+
+The options in this section configure the experimental Feature Toggle Admin Page feature, which is enabled using the `featureToggleAdminPage` feature toggle. **Use at your own risk.**
```

Comment:
I can't find any other places in the docs were we use the phrase "Use at your own risk."  Currently consulting with the team on if we have a standard caution for experimental features.

Reply:
```suggestion
The options in this section configure the experimental Feature Toggle Admin Page feature, which is enabled using the `featureToggleAdminPage` feature toggle. Projects and features in the experimental stage are not meant to be used in production environments, and the risks are unknown.
```

Reply:
Another option:
```suggestion
The options in this section configure the experimental Feature Toggle Admin Page feature, which is enabled using the `featureToggleAdminPage` feature toggle. Grafana Labs offers support on a best-effort basis, and breaking changes might occur prior to the feature being made generally available.
```